Move contributing.md (#2030)
WordPress/openverse: documentation/general/contributing.md

# Welcome

Thank you for your interest in Openverse! We're so excited to bring new
contributors into the fold in Openverse, WordPress and FOSS in general. This
document is a set of guidelines to help you contribute to this project.

## Code of Conduct

You should read and agree to abide by the
[code of conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md)
before contributing to WordPress projects. This applies to all Openverse
repositories because Openverse is a WordPress Foundation project.

## Keep in touch

Don't hesitate to ask for help; if you're stuck, we're here for you! You can
ping us via any of our
[communication channels](https://github.com/WordPress/openverse#keep-in-touch).

## Code and code-related contributions

### Get started

Detailed help for contributing code can be found in the
[developer documentation](https://docs.openverse.org/), which also includes the
following subfolders.

The following resources are preferred reading for starting your Openverse
contribution journey in the code and code-related space.

> - [GitHub contribution practices](/meta/github_contribution_practices.html)
> - [Dev flow](/meta/dev_flow.html)
> - [General setup guide](/general/general_setup.html)
> - [Quickstart guide](/general/quickstart.html)

Reviewer: Saw some more warnings in the documentation build logs. 
<img width="779" alt="Screenshot 2023-05-07 at 12 08 16 PM" src="https://user-images.githubusercontent.com/16580576/236665668-23cabac9-702b-471a-9248-4cf447542d1c.png">

Apparently the links should point to the raw Markdown files and not HTML pages.

```suggestion
- [GitHub contribution practices](/meta/github_contribution_practices.md)
- [Dev flow](/meta/dev_flow.md)
- [General setup guide](/general/general_setup.md)
- [Quickstart guide](/general/quickstart.md)
```

# Welcome

Thank you for your interest in Openverse! We're so excited to bring new
contributors into the fold in Openverse, WordPress and FOSS in general. This
document is a set of guidelines to help you contribute to this project.

## Code of Conduct

> You should read and agree to abide by the [code of conduct](CODE_OF_CONDUCT.md)

Reviewer: ```suggestion
You should read and agree to abide by the [code of conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md)
```

before contributing to WordPress projects. This applies to all Openverse
repositories because Openverse is a WordPress Foundation project.

## Keep in touch

Don't hesitate to ask for help; if you're stuck, we're here for you! You can

> ping us via any of our [communication channels](README.md#keep-in-touch).

Reviewer: ```suggestion
ping us via any of our [communication channels](https://github.com/WordPress/openverse#keep-in-touch).
```

## Code and code-related contributions

### Get started

Detailed help for contributing code can be found in the
[developer documentation](https://docs.openverse.org/), which also includes the
following subfolders.

- [Storybook](https://docs.openverse.org/storybook/) (UI components)

> - [Tailwind Config Viewer](https://docs.openverse.org/tailwind/) (design tokens)

Reviewer: This is covered in the root of the documentation (https://github.com/WordPress/openverse/blob/main/documentation/index.md#related-documentation), so this section can be deleted.

The following resources are preferred reading for starting your Openverse
contribution journey in the code and code-related space.

> - [GitHub contribution practices](https://docs.openverse.org/meta/github_contribution_practices.html)
> - [Dev flow](https://docs.openverse.org/meta/dev_flow.html)
> - [General setup guide](https://docs.openverse.org/general/general_setup.html)
> - [Quickstart guide](https://docs.openverse.org/general/quickstart.html)

Reviewer: ```suggestion
- [GitHub contribution practices](/meta/github_contribution_practices.html)
- [Dev flow](/meta/dev_flow.html)
- [General setup guide](/general/general_setup.html)
- [Quickstart guide](/general/quickstart.html)
```

You can use the
[New Source Suggestion for Openverse](https://github.com/WordPress/openverse/issues/new?assignees=&labels=%F0%9F%9A%A6+status%3A+awaiting+triage%2C%F0%9F%A7%B9+status%3A+ticket+work+required%2C%E2%98%81%EF%B8%8F+provider%3A+any&template=new_source_suggestion.yml&title=%3CSource+name+here%3E)
issue template to submit whatever sources you find.

### Feedback

Feedback like reporting bugs and missing features is big help for Openverse. We
want Openverse to be a bug-free app with great user experience and providing
feedback is a way to put any shortcomings on our radar and get them resolved.

You can report bugs, request features and see all other forms of feedback we
would like to receive by
[submitting a new issue](https://github.com/WordPress/openverse/issues/new/choose)
on GitHub. All are welcome to write issues and the Openverse maintainers have
deep gratitude for those who do. Note that you will need a GitHub account to
create new issues.

You can also provide feedback via any of our other

> [communication channels](README.md#keep-in-touch).

Reviewer: ```suggestion
[communication channels](https://github.com/WordPress/openverse#keep-in-touch).
```
